pkg/extensions/search/resolver.go

```diff
@@ -216,39 +216,45 @@ func globalSearch(repoList []string, name, tag string, olu common.OciLayoutUtils
 			log.Error().Err(err).Msgf("can't find latest updated tag for repo: %s", repo)
 		}
 
-		tagsInfo, err := olu.GetImageTagsWithTimestamp(repo)
+		repoInfo, err := olu.GetExpandedRepoInfo(repo)
 		if err != nil {
-			log.Error().Err(err).Msgf("can't get tags info for repo: %s", repo)
+			log.Error().Err(err).Msgf("can't get repo info for repo: %s", repo)
 
 			continue
 		}
 
-		repoInfo, err := olu.GetExpandedRepoInfo(repo)
+		manifests, err := olu.GetImageManifests(repo)
```

Comment:
Wondering why we need this, don't we have all the needed data in `repoInfo.Manifests`?

Reply:
It is the same reason why we needed tagsInfo in the first place. repoInfo has a list of manifests with encoded digests, while olu.GetImageManifests(repo) gives us a list of manifests that have full digests ( i.e. digests that include the algorithm prefix).